include/sparrow/types/data_type.hpp

```diff
@@ -617,7 +618,9 @@ namespace sparrow
         float32_t,
         float64_t,
         std::string,
+        std::string_view,
         std::vector<byte_t>,
+        sparrow::sequence_view<const byte_t>,
```

Comment:
`all_base_types_t`  should contain types for the inner value_type only, not types used in the const_reference type.,